packages/frontend/src/components/Explorer/FormatModal/index.tsx

```diff
@@ -97,7 +98,9 @@ export const FormatModal = () => {
             opened={isOpen}
             onClose={handleClose}
             title={
-                <Title order={4}>Format metric "{getItemLabel(metric)}"</Title>
+                <Title order={4}>
+                    Format metric "{getItemLabelWithoutTableName(metric)}"
```

Comment:
> remove the metric name from the modal title

Ticket mentioned no metric name at all but I assume you checked this with Katie.